Make-a-pull-request.md

```diff
@@ -106,11 +106,9 @@ Once your feature is ready, you can open a pull request (PR)!
 
 * Click "Create pull request".
 
-* If you have already completed 2 pull requests and been added as a collaborator to the project, you should also add a changelog label.  If you are a new contributor, you don't have permission to do this.  Don't worry! Oppiabot will automatically ask someone to do it for you.
-
 * Oppiabot will check that you filled out the PR description correctly. If you didn't, it will leave a comment explaining what you need to do to fix it. More details on the kinds of comments Oppiabot leaves are coming soon.
 
-* GitHub will automatically assign reviewers, and Oppiabot will assign the issue's "owner" (from the changelog label) to do the first review.
+* GitHub will automatically assign reviewers, and Oppiabot will assign them as codeowners to do the first review.
```

Comment:
I'm reading this as "Oppiabot assigns them as code owner"

Should we use "add code owners as reviewers" and "some of them as assignees to the PR for the first review"? (More around the Github "Reviewers" and "Assignees" keywords.)
![image](https://user-images.githubusercontent.com/16653571/225878005-2277cfae-9763-419b-a074-bc4eb302945d.png)


Something around this:
```
GitHub will automatically add codeowners as reviewers, and after that Oppiabot will add some of the reviewers as assignees for the first review.
```

Reply:
Thanks @DubeySandeep for taking a look! Actually the behaviour here has changed a bit -- see https://github.com/oppia/oppiabot/pull/314#issuecomment-1454792798 . So I think the current sentence is actually correct and the proposed revision is not. Could you please confirm?